backend/EasySpeak.Core/EasySpeak.Core.Common/DTO/Lesson/FiltersRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using EasySpeak.Core.DAL.Entities.Enums;
+using Newtonsoft.Json;
+
+namespace EasySpeak.Core.Common.DTO.Lesson;
+
+public class FiltersRequest
+{
+    [JsonProperty(Required = Required.Default)]
```

Comment:
You can required keyword or JsonRequired attribute instead of this

Reply:
fixed